static/scss/style.scss

```diff
@@ -45,7 +45,7 @@
   background-color: #bee2d4;
 }
 
-.no-display {
+.no-display, .home, .active-pg {
```

Comment:
.home is maybe too likely to be used somewhere else by accident. What about `#page-header .home`?